client/src/components/App.jsx

```diff
@@ -1,17 +1,61 @@
+/* eslint-disable camelcase */
 import React from 'react';
+import axios from 'axios';
 import './App.css';
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      productId: 47425,
+      displayProduct: [],
+    };
+    this.formatBody = this.formatBody.bind(this);
+
+  }
+
+  formatBody(method, apiRoute, params, data) {
+
+    let bodyObj = {
+      method: method,
+      url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp${apiRoute}`,
+      data: data,
+      params: params,
+      headers: {Authorization: ''}
+    };
+
+
```

Comment:
There are some extra spaces all over the place. We can remove that by using any **_formatter_** or manually.